src/app/settings/components/settingsPane.tsx

```diff
@@ -128,13 +132,26 @@ export default class SettingsPane extends React.Component<SettingsPaneProps & Se
                                 onRemoveListItem={this.onRemoveListItem}
                             />
                         </section>
+                        <section aria-label={context.t(ResourceKeys.settings.theme.headerText)}>
+                            <h3 role="heading" aria-level={1}>{context.t(ResourceKeys.settings.theme.headerText)}</h3>
+                            <Toggle
+                                onText={context.t(ResourceKeys.settings.theme.darkTheme)}
+                                offText={context.t(ResourceKeys.settings.theme.lightTheme)}
+                                onChange={this.setTheme}
+                                checked={this.props.theme === Theme.dark}
+                            />
+                        </section>
                         {this.renderConfirmationDialog(context)}
                     </Panel>
                 )}
             </LocalizationContextConsumer>
         );
     }
 
+    private readonly setTheme = (ev: React.MouseEvent<HTMLElement>, darkTheme: boolean) => {
+        this.props.onSetTheme(darkTheme ? Theme.dark : Theme.light);
```

Comment:
It's my thought that we could probably follow the "save before action takes into place" pattern in this pane. We should probably run through this decision with Mia as well as include this button in the settings pane.

Reply:
I talked to Mia. She is ok putting the toggle in this page.
My suggestion is to make user click save before the changes apply. I looked at ADB, their language preference settings follows the same pattern. I think it also aligns with the accesibility guidelines a bit better.

Reply:
Have implemented as described above while moving to context-based theme changes.